src/styles/pages/appointments/Create.module.css

```diff
@@ -0,0 +1,40 @@
+.createContainer {
```

Comment:
nit: Can we prefix with `createAppointment` for all the classes, just so it is more specific